models/intermediate/int_pendo__feature_info.sql

```diff
@@ -1,7 +1,7 @@
 with feature as (
 
     select 
-        {{ dbt_utils.star(from=ref('int_pendo__latest_feature'), except='LATEST_FEATURE_INDEX' if target.type == 'snowflake' else 'latest_feature_index') }}
+        {{ dbt_utils.star(from=ref('int_pendo__latest_feature'), except='latest_feature_index') }}
```

Comment:
Curious why the `guid_id` in the `pendo__guide` model failed because it was not an array, but this one succeeded? 🤔 

Would you be able to update this to be enclose this exclusion in an array for consistency.

Reply:
yah this one succeeded because we are not excluding `latest_feature_index` for ambiguous column reasons -- we're just removing it because it is an unnecessary column (a vestige of the row_number window function we use to remove old historical records [here](https://github.com/fivetran/dbt_pendo/blob/main/models/intermediate/latest_history/int_pendo__latest_feature.sql)). i'm not sure why i decided to do that for features only (and not guide, page, etc) however -- it seems like i forgot to roll that out elsewhere. flagging in case that's something we want to correct as well... @fivetran-joemarkiewicz

Reply:
Dug into this with @fivetran-reneeli and the exclusions MUST be enclosed in an array. We are good to not exclude the latest_index in the other models.